gcpy/benchmark/modules/run_1yr_fullchem_benchmark.py

```diff
@@ -15,19 +15,15 @@
 
 To generate benchmark output:
 
-    (1) Copy the gcpy/benchmark/run_benchmark.py script and the
-        1yr_fullchem_benchmark.yml file anywhere you want to generate
-        output plots and tables.
+    (1) Copy the file gcpy/benchmark/config/1yr_fullchem_benchmark.yml
+        to a folder of your choice.
 
-    (2) Edit the 1yr_fullchem_benchmark.yml to point to the proper
-        file paths on your disk space.
+    (2) Edit the 1yr_tt_benchmark.yml to select the desired options
```

Comment:
I think tt here should be fullchem

Reply:
@msulprizio: Now fixed in commit 3b380be2.